logger.py

```diff
@@ -30,6 +30,12 @@ def display_message(is_debug, category, user_message, add_crlf=True):
         formatted_message = ("[%s] [%s] %s" %
                              (time.strftime("%Y/%m/%d %I:%M:%S"),
                               category.rjust(category_width), user_message))
+        
+        #Print the output also to a logfile
+        logfile = open("hb-downloader_" + time.strftime("%Y-%m-%d") + ".log", 'a')
+        print(formatted_message, file = logfile)
+        logfile.close()
```

Comment:
A couple of nitpicks:

* You're not handling `add_crlf` being `False`
* You're opening and closing the file every single time, which is quite expensive; fixing this seems non-trivial if you _do_ want to be neat and call `.close`...